src/main/resources/init.sql

```diff
@@ -0,0 +1,15 @@
+DROP SCHEMA IF EXISTS CAESAR;
+CREATE SCHEMA CAESAR CHARACTER SET UTF8;
+USE CAESAR;
+
+
+CREATE USER IF NOT EXISTS 'admin'@'localhost' IDENTIFIED BY '1234%';
+GRANT ALL PRIVILEGES ON *.* TO 'admin'@'localhost';
+
+CREATE USER IF NOT EXISTS 'db_admin'@'localhost' IDENTIFIED BY '1234%';
+GRANT ALL PRIVILEGES ON CAESAR.* TO 'admin'@'localhost';
```

Comment:
Please generate more complicated password (16 characters long, having caps and lower case, as well as special characters and numbers) for the user and set it to some property in application.properties.